src/_data/LiveData.js

```diff
@@ -56,15 +56,14 @@ export const changeLanguage = langCode => {
 };
 
 const initUnauthorized = async () => {
-
-};
-
-const initAuthorized = async (authData, store) => {
     api.getActiveSymbolsFull();
     api.getTradingTimes(new Date());
     api.getAssetIndex();
     api.getServerTime();
     api.getCandlesForLastNDays('R_100', 30);
```

Comment:
@borisyankov do you remember why do we move all api call after authorize?
Does it improve initial loading time?